boofuzz/blocks/checksum.py

```diff
@@ -28,6 +28,9 @@ class Checksum(primitives.BasePrimitive):
     the algorithm parameter.
 
     The length field is only necessary for custom algorithms.
+    When using your own provided custom checksum function return is the calculated crc of the data.
+
+    Function signature:  <function_name>(data_bytes, starting_value in hex. Returns a number.
```

Comment:
A custom checksum function will only be called with one argument, the data in bytes type. The function then needs to return the checksum also as a bytes type, not a number/integer.
Also there is a parentheses missing.